Source/Synthesis/FieldTypes/FileField.cs

```diff
@@ -93,8 +93,8 @@ public override bool HasValue
 		{
 			get
 			{
-				if (InnerField == null) return false; 
-				return !(MediaItemId == (ID)null || MediaItemId.IsNull);
+				if (InnerField == null) return false;
+				return !(string.IsNullOrEmpty(InnerField.Value));
```

Comment:
Hi @rkuchler - this change looks like it'd make sense to me. What sort of value would ContentHub be putting in this field? Is it a URL? 

I want to add an example to my testing suite.